scripts/apps/authoring/metadata/PlacesService.ts

```diff
@@ -23,25 +52,39 @@ export default function PlacesServiceFactory(api, features, metadata) {
 
     class PlacesService implements IPlacesService {
 
-        search(name: string, lang: string, format: 'geonames' | 'dateline') {
-            return this.searchGeonames(name, lang, format === 'dateline')
-                .then((geonames) => format === 'geonames' ? geonames : geonames.map(geonameToCity))
-                .catch(() =>
-                    format === 'dateline'
-                        ? this.searchCities(name)
-                        : Promise.reject(), // can't handle geonames via cities
-                );
+        /**
+         * Search for dateline
+         *
+         * it will use geonames if available, cities cv otherwise
+         *
+         * @param name
```

Comment:
Why is `@params` included in the comment? The type signature is already in the interface & implementation.

Can you move the comment to the interface? It's easier to find it that way. If one needs to consume the service and not modify it, reading the implementation shouldn't be required.

Specify what is supported as `name` argument. Can it be a street name, a post code, the coordinates?